### ISSUE TITLE
Jeroen's usename is now jeroen, not jeroenooms

### ISSUE THREAD

**Issue by AmeliaMN** [r-lib/gh#94]

**Comment by AmeliaMN** [comment]
I was baffled by the responses I was getting back from
```
j_repos <- gh("/users/:username/repos", username = "jeroenooms")
```
because I know Jeroen has lots of repos! Turns out the README is just using his old username.

**Comment by gaborcsardi** [comment]
Well, that's what you get if you change your GH username. :)

**Comment by gaborcsardi** [comment]
Closed in 61a5b550eda5578c5a2192545be8e83a687eb3bf
